packages/discord.js/src/structures/CategoryChannel.js

```diff
@@ -1,19 +1,21 @@
 'use strict';
 
 const GuildChannel = require('./GuildChannel');
+const CategoryChannelChildManager = require('../managers/CategoryChannelChildManager');
 
 /**
  * Represents a guild category channel on Discord.
  * @extends {GuildChannel}
  */
 class CategoryChannel extends GuildChannel {
-  /**
-   * Channels that are a part of this category
-   * @type {Collection<Snowflake, GuildChannel>}
-   * @readonly
-   */
-  get children() {
-    return this.guild.channels.cache.filter(c => c.parentId === this.id);
+  constructor(guild, data, client) {
+    super(guild, data, client);
+
+    /**
+     * A manager of the channels belonging to this category
+     * @type {CategoryChannelChildManager}
+     */
+    this.children = new CategoryChannelChildManager(this);
```

Comment:
This could be a getter, no need for the extra memory overhead.

Reply:
A getter for a manager? Do we do that anywhere else?

Reply:
https://github.com/discordjs/discord.js/blob/11e5e5ac5b70138f56332eb3e61a42443670b0de/packages/discord.js/src/client/Client.js#L172-L183

Reply:
Not sure what difference this makes since the cache for this manager is a getter but sure, I'll change that